netlify/functions/routines.ts

```
@@ -0,0 +1,32 @@
+      if (!userId) return { statusCode: 400, body: 'Missing userId' };
+      const routines = await storage.getUserRoutines(parseInt(userId));
+      return { statusCode: 200, body: JSON.stringify(routines) };
+    }
+    if (event.httpMethod === 'POST') {
+      const body = event.body ? JSON.parse(event.body) : {};
+      const routineData = insertRoutineSchema.parse(body);
+      const routine = await storage.createRoutine(routineData);
+      return { statusCode: 200, body: JSON.stringify(routine) };
+    }
+    if (event.httpMethod === 'PUT') {
+      const id = event.queryStringParameters && event.queryStringParameters.id;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)